src/components/ToDo/styles/ToDoCard.styled.ts

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components'
+import iconcheck from '../../../assets/todo/icon-check.svg'
+import iconcheckblack from '../../../assets/todo/icon-check-black.svg'
+
+export const Rectangle = styled.div<{ isDarkMode: boolean }>`
+  height: 60px;
+  background-color: ${(props) => (props.isDarkMode ? 'hsl(235, 24%, 19%)' : 'hsl(0, 0%, 98%)')};
+  border-radius: 0px;
+  box-shadow: ${(props) => (props.isDarkMode ? '2px 2px 4px #000000' : '')};
```

Comment:
this could be done with `${(props) => (props.isDarkMode && 'box-shadow: 2px 2px 4px #000;')}` instead so it only appears when isDarkMode is true